server/schedule/operator/operator.go

```diff
@@ -247,6 +246,15 @@ func (o *Operator) Step(i int) OpStep {
 	return nil
 }
 
+// getStepStartTime returns the start time of the i-th step.
+func (o *Operator) getStepStartTime(step int32) time.Time {
+	startTime := o.GetStartTime()
+	if 0 < step && int(step-1) < len(o.steps) {
```

Comment:
I don't get this line, why do we need to compare `step` with `len(o.steps)` after it's smaller than 0?